communication/README.md

```diff
@@ -122,7 +122,7 @@ try to post on discuss.kubernetes.io or if you think you found a bug, please [fi
 
 ### Office Hours
 
-Office hours are held once a month. Please refer to [the docs][office-hours]
+Office hours are held once a month. Please refer to [events/office-hours](../events/office-hours.md)
```

Comment:
The link is there as a reference link that's located at the bottom, but it's missing a `-` >_<
```
[slack moderators]: ./moderators.md#slack
[#kubernetes-dev]: https://app.slack.com/client/T09NY5SBT/C09R23FHP
[#sig-contribex]: https://app.slack.com/client/T09NY5SBT/C1TU9EB9S
[#office-hours]: https://app.slack.com/client/T09NY5SBT/C6RFQ3T5H
[office hours]: /events/office-hours.md
```

Reply:
It's an even smaller change then :sweat_smile:

Reply:
Yup! Very easy to overlook when writing the markdown directly lol.